lib/matplotlib/backend_managers.py

```diff
@@ -74,6 +73,43 @@ def __init__(self, canvas):
         self.keypresslock = widgets.LockDraw()
         self.messagelock = widgets.LockDraw()
 
+        if figure:
+            self.figure = figure
```

Comment:
This may be nit-picking, but I would put the initialization of `self._figure` right above this, and instead of using the property I would use the setter.  To me, eliminating that indirection would make the code more readable with no cost.